ui/lib/config-ui/addon/routes/messages/message/edit.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+import { hash } from 'rsvp';
+
+export default class MessagesMessageEditRoute extends Route {
+  @service store;
+
+  async getMessages(authenticated = true) {
+    try {
+      return await this.store.query('config-ui/message', {
+        authenticated,
+      });
+    } catch {
+      return [];
+    }
+  }
```

Comment:
Async/await is not needed here since you aren't accessing the result of the resolved promise:
```suggestion
  getMessages(authenticated = true) {
    return this.store.query('config-ui/message', { authenticated }).catch(() => []);
  }
```